api/config/service/authn/v1/authn.proto

```diff
@@ -14,6 +14,9 @@ message OIDC {
   string redirect_url = 4 [ (validate.rules).string = {min_bytes : 1} ];
 
   repeated string scopes = 5;
+
+  string subject_claim_name = 6;
+  string groups_claim_name = 7;
```

Comment:
Should we do something like this to indicate that this is overriding the default? Also can you add a comment with more detail description of the behavior so it's more clear to readers, i don't think will be able to be expressive enough with just the name of the variable.

as an example maybe something like this?
```
  string override_subject_claim_name= 6;
  string override_groups_claim_name = 7;
```